tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -985,11 +998,177 @@ public void SetDefaultExtensions(string recordingId = null)
+                    new BodyKeySanitizer("$..source"),
+                    // new BodyKeySanitizer("$..resourceLocation"), disabled, not a secret?
+                    new HeaderRegexSanitizer("Location"),
+                    new BodyKeySanitizer("$..to"),
+                    new BodyKeySanitizer("$..from"),
+                    new HeaderRegexSanitizer("subscription-key"),
+                    new BodyKeySanitizer("$..outputDataUri"),
+                    new BodyKeySanitizer("$..inputDataUri"),
+                    new BodyKeySanitizer("$..containerUri"),
+                    new BodyKeySanitizer("$..sasUri"),
+                    new BodyRegexSanitizer(regex: "(?:(sv|sig|se|srt|ss|sp)=)(?<secret>[^&\\\"\\s]*)", groupForReplace: "secret"),
+                    new BodyKeySanitizer("$..id"),
+                    new BodyKeySanitizer("$..token"),
+                    new BodyKeySanitizer("$..appId"),
+                    new BodyKeySanitizer("$..userId"),
+                    // "name" I think this one is too generic, we should probably drop it?
+                    new BodyKeySanitizer("$..id"),
+                    new BodyKeySanitizer("$..storageAccount"),
+                    new BodyKeySanitizer("$..resourceGroup"),
+                    new BodyKeySanitizer("$..guardian"),
+                    new BodyKeySanitizer("$..scan"),
+                    new BodyKeySanitizer("$..catalog"),
+                    new BodyKeySanitizer("$..lastModifiedBy"),
+                    new BodyKeySanitizer("$..managedResourceGroupName"),
+                    // new BodyKeySanitizer("$..friendlyName"), disabled, not a secret?
+                    new BodyKeySanitizer("$..createdBy"),
+                    new BodyKeySanitizer("$..tenantId"),
+                    new BodyKeySanitizer("$..principalId"),
+                    new BodyKeySanitizer("$..clientId"),
+                    new BodyKeySanitizer("$..credential"),
+                    new HeaderRegexSanitizer("SupplementaryAuthorization"),
+                    new BodyKeySanitizer("$.key"),
+                    new BodyKeySanitizer("$.value[*].key"),
+                    new UriRegexSanitizer(regex: "sig=(?<sig>[^&]+)", groupForReplace: "sig"),
+                    // new HeaderRegexSanitizer("x-ms-encryption-key"), dup
+                    new HeaderRegexSanitizer("x-ms-rename-source"),
+                    new HeaderRegexSanitizer("x-ms-file-rename-source"),
+                    new HeaderRegexSanitizer("x-ms-copy-source"),
+                    new HeaderRegexSanitizer("x-ms-copy-source-authorization"),
+                    new HeaderRegexSanitizer("x-ms-file-rename-source-authorization"),
+                    new HeaderRegexSanitizer("x-ms-encryption-key-sha256"),
+                    new BodyKeySanitizer("$..uploadUrl"),
+                    new BodyKeySanitizer("$..logLink"),
+                    new HeaderRegexSanitizer("aeg-sas-token"),
```

Comment:
nit: would it be worth sorting these based on the sanitizer type, and potentially alphabetically within each type?

Reply:
I'll do it now before I assign them internal ids 👍

Reply:
They are now organized by sanitizer type, and the default rules list has gaps if we need to insert some additional bodyregex vs bodykey vs generalregex sanitizers.